zerver/lib/redis_utils.py

```diff
@@ -3,6 +3,8 @@
 from django.conf import settings
 
 import redis
+from redis import StrictRedis
 
 def get_redis_client():
+    # type: () -> StrictRedis
```

Comment:
might as well make the next line just use `StrictRedis` if you're going to import it (you can also use e.g. redis.StrictRedis in the annotations)